utilities/transactions/write_prepared_txn.cc

```diff
@@ -51,9 +51,21 @@ Status WritePreparedTxn::PrepareInternal() {
 }
 
 Status WritePreparedTxn::CommitWithoutPrepareInternal() {
-  // TODO(myabandeh) Implement this
-  throw std::runtime_error("Commit not Implemented");
-  return Status::OK();
+  return CommitBatchInternal(GetWriteBatch()->GetWriteBatch());
+}
+
+Status WritePreparedTxn::CommitBatchInternal(WriteBatch* batch) {
+  const bool disable_memtable = true;
+  const uint64_t no_log_ref = 0;
+  uint64_t seq_used;
+  auto s = db_impl_->WriteImpl(write_options_, batch, nullptr, nullptr,
+                               no_log_ref, !disable_memtable, &seq_used);
+  uint64_t& prepare_seq = seq_used;
```

Comment:
why reference? It is just a primitive int type.

Reply:
I needed an alias just to improve code readability and reference is the way to do. Did not mean to improve performance.

Reply:
yeah, so why not remove the reference.

Reply:
The reference shows the intention of the developer of using it just as an alias.